TAScheduler/views_methods.py

```diff
@@ -582,6 +623,26 @@ def getInstrCrseAsgmts(self):
     def getInstrLecAsgmts(self):  # new
         return Lecture.objects.filter(instructor=self.database)
 
+    def editContactInfo(self, **kwargs):
+        user = self.database.user  # Accessing the User object
+        for key, value in kwargs.items():
+            if hasattr(user, key):
+                setattr(user, key, value)
+        user.save()
+
+    def getContactInfo(self):
+        user = self.database.user
+        return {
+            'first_name': user.first_name,
+            'last_name': user.last_name,
+            'email_address': user.email_address,
+            'home_address': user.home_address,
+            'phone_number': user.phone_number
+        }
+
+    def getAllUserAssignments(self):
+        return InstructorToCourse.objects.filter(instructor=self.database)
```

Comment:
I think might need to include Section assignments as well

Reply:
I could, but i dont know if thats necessary tbh. anyone else have insight on this?

Reply:
I originally intended all users to have this method and to return something like 
<img width="514" alt="Screen Shot 2023-12-11 at 9 05 17 PM" src="https://github.com/KevinMonahanUWM/SchedulerApp/assets/123400843/6a90079b-fa6e-44f8-8357-ddb351b9333f">

I would say you might as well deal with getting the sections in this method here so you don't have to later. If we don't need sections when calling the method then we don't need to use it but it may be nice to have ig.

Reply:
done